.github/workflows/test.yml

```diff
@@ -55,6 +55,27 @@ jobs:
         if: runner.os == 'Linux' && matrix.app_version == 'stable'
         env:
           NEOVIM_PATH: ${{ steps.vim.outputs.executable }}
+      - name: Set up Lua
+        uses: leafo/gh-actions-lua@v9
+        with:
+          luaVersion: "luajit-2.1.0-beta3"
+        if: runner.os == 'Linux' && matrix.app_version == 'stable'
+      - name: Set up Luarocks
+        uses: leafo/gh-actions-luarocks@v4
+        with:
+          luaVersion: "5.1"
+        if: runner.os == 'Linux' && matrix.app_version == 'stable'
+      - name: Install Busted
+        run: |
+          luarocks install busted
+          luarocks install luafilesystem
```

Comment:
could you merge the 2 "install" in a single line?

Reply:
Just notice this failed, so maybe can't be one line? Hadn't bothered tested it locally since it seemed sane, but will fix tomorrow.

Reply:
https://github.com/luarocks/luarocks/issues/1359

Reply:
that being said, if contents is into another test.yml, 2 lines is fine really.

Reply:
or `luarocks install busted && luarocks install busted` lol